README.md

```diff
@@ -51,6 +51,10 @@ If you want to use S3 storage for the sccache cache, you need to set the `SCCACH
 
 Set `SCCACHE_REDIS` to a [Redis](https://redis.io/) url in format `redis://[:<passwd>@]<hostname>[:port][/<db>]` to store the cache in a Redis instance.
 
+To use [GCS](https://cloud.google.com/storage/), you need to set the `SCCACHE_GCS_BUCKET` environment variable to the name of the GCS bucket.
+If you're using authentication, set `SCCACHE_GCS_KEY_PATH` to the location of your JSON service account credentials.
+By default, SCCACHE on GCS will be read-only. To change this, set `SCCACHE_GCS_RW_MODE` to either `READ_ONLY` or `READ_WRITE`.
```

Comment:
Our vision for this is to populate the cache using bots, which are given a key.  Anyone else should be able to just point sccache at the public bucket and use it as a read-only cache without having to use any credentials.

However, if you think that's not going to be the common case, I'd be happy to discuss how to accommodate other scenarios. One option would be to default to READ_WRITE if credentials are provided, and READ_ONLY otherwise. What do you think?